tools/docker/development/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.11
+WORKDIR /rathena
+RUN apk add --no-cache wget git cmake make gcc g++ gdb zlib-dev mariadb-dev ca-certificates linux-headers bash valgrind netcat-openbsd
+RUN wget https://raw.githubusercontent.com/eficode/wait-for/v2.2.2/wait-for -O /bin/wait-for && chmod +x /bin/wait-for
```

Comment:
I'm not fan of this, better download and add the script as an asset.
Like this it look quite unsafe. (yes it's probably only meant for dev but still, why do we need to download this ?)

Reply:
me neither. that's the part I factored out from the old docker compose setup. Everything under `docker/development` is. The things I propose to add are everything but those files.